### PR TITLE
[SCAL-92011] - Added breadcrums depends on routes change

### PR DESCRIPTION
**Pull request by harsh-ts**
Ticke Id: https://thoughtspot.atlassian.net/browse/SCAL-92011

### CODE REVIEW

File: docs/src/assets/styles/variables.scss [hunk 1]
```diff
@@ -63,7 +63,7 @@ $font-family-doc: Optimo-Plain;
 $font-family-back-button: Optimo-Plain;
 $width-back-button: 32px;
 $height-back-button: 32px;
-$margin-top-mobile: 55px;
+$margin-top-mobile: 58px;
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/assets/styles/variables.scss:66]: this looks funky, can we have some sort of standardisation here. we can get some idea from radiant-styles/_base.scss

File: docs/src/components/Breadcrums/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                    <li key={`${name}-${href}`}>
+                        {href ? <a href={href}>{name}</a> : name}
+                    </li>
+                ))}
+            </ul>
+        </div>
+    );
+};
+
+export default Breadcrums;
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/components/Breadcrums/index.tsx:31]: can we use named export here, easy to add more export in future

File: docs/src/assets/styles/variables.scss (hunk 1 above)
Comment (harsh-ts) [reply docs/src/assets/styles/variables.scss:66]: The style is fixed, as we added some component in search bar we have increased it height. and we are using the vairable inorder to have single source

File: docs/src/components/Breadcrums/index.tsx (hunk 2 above)
Comment (harsh-ts) [reply docs/src/components/Breadcrums/index.tsx:31]: Its a single component, If we want to add different component then we will create seperate file for that (Thats the structure I am following in this repo). Let me know your thoughts on this

File: docs/src/components/Breadcrums/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+type BreadcrumsProps = {
+    pageid?: string;
+    breadcrumsData: any;
+};
+
+const Breadcrums: React.FC<BreadcrumsProps> = (props: BreadcrumsProps) => {
+    const breadcrums = getBreadcrumsPath(props.breadcrumsData, props.pageid);
+
+    if (!breadcrums.length) {
+        return <div />;
```
[the remark this answers is not in the document]
Comment (harsh-ts) [reply docs/src/components/Breadcrums/index.tsx:14]: Will make changes

**Review by kailash-choudhary-thoughtspot — APPROVED** [review]

**Pull request merged**